charter.md

```diff
@@ -1,56 +1,113 @@
+
+<table>
+    <thead>
+        <tr>
+            <th>IRC nick</th>
+            <th>Github</th>
+            <th>Project</th>
+        </tr>
+    </thead>
+    <tbody>
+        <tr>
+            <td>Attila</td>
+            <td><a href="https://github.com/attilamolnar">attilamolnar</a></td>
+            <td><a href="http://www.inspircd.org/">InspIRCd</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>dan-</td>
+            <td><a href="https://github.com/DanielOaks">DanielOaks</a></td>
+            <td><a href="https://github.com/mammon-ircd/mammon">mammon</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>DarthGandalf</td>
+            <td><a href="https://github.com/DarthGandalf">DarthGandalf</a></td>
+            <td><a href="http://wiki.znc.in/ZNC">ZNC</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>dx</td>
+            <td><a href="https://github.com/dequis">dequis</a></td>
+            <td><a href="https://www.bitlbee.org/">bitlbee</a>/<a href="https://irssi.org/">irssi</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>grawity</td>
+            <td><a href="https://github.com/grawity">grawity</a></td>
+            <td>unaffiliated</td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>jwheare</td>
+            <td><a href="https://github.com/jwheare">jwheare</a></td>
+            <td><a href="https://www.irccloud.com/">IRCCloud</a></td>
+            <td>Chair</td>
+        </tr>
+        <tr>
+            <td>M2Ys4U</td>
+            <td><a href="https://github.com/M2Ys4U">M2Ys4U</a></td>
+            <td><a href="https://kiwiirc.com/">Kiwi IRC</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>TingPing</td>
+            <td><a href="https://github.com/TingPing">TingPing</a></td>
+            <td><a href="https://hexchat.github.io/">HexChat</a></td>
+            <td></td>
+        </tr>
+        <tr>
+            <td>NoOneButMe</td>
+            <td><a href="https://github.com/zadr">zadr</a></td>
+            <td><a href="http://colloquy.info/">Colloquy</a></td>
+            <td></td>
+        </tr>
+    </tbody>
+</table>
+
+The chair has ultimate responsibility for moving discussions forward and will make decisions in cases where consensus isn't possible, including returning proposals to the drawing board where necessary.
+
+## Project resources & Contribution
+
+The main project resources are:
+
+ * The [ircv3/ircv3-specifications](https://github.com/ircv3/ircv3-specifications) Github repository and issue tracker.  This is the
+   official repository containing the latest version of the specifications.  Changes may be submitted by pull request. Suggestions may be made by opening an issue.
+
+ * The [#ircv3](irc://irc.freenode.net/ircv3) channel at [irc.freenode.net](irc://irc.freenode.net/ircv3).  This is the main discussion space for the working group.
```

Comment:
This should render fine on the website, looks like Github Markdown just doesn't like the links.